docs/serving/samples/hello-world/helloworld-go/README.md

```diff
@@ -60,13 +60,13 @@ recreate the source files from this folder.
    # Use the offical Golang image to create a build artifact.
    # This is based on Debian and sets the GOPATH to /go.
    # https://hub.docker.com/_/golang
-   FROM golang as builder
+   FROM golang:1.12 as builder
```

Comment:
Why 1.12 explicitly?

Reply:
The purpose of this PR is consolidating README files, all these changes are due to inconsistency between source file and README. Preferably I would pick the one comes later, but it's not doable as they all lost history in github UI due to recent movement. So I just blindly picked one that I thought being later

Reply:
for example this line comes from this file:
https://github.com/knative/docs/blob/52837ad26ad4cd2e3e93a5c17e8a32941d1e6a23/docs/serving/samples/hello-world/helloworld-go/Dockerfile#L4

Reply:
Thanks for clarifying.